gnovm/cmd/benchops/stats.go

```diff
@@ -53,6 +53,10 @@ func stats(binFile string) {
 					opName = bm.StoreCodeString(record[1])
 				}
 
+				if record[2] != 0 {
+					opName = bm.NativeCodeString(record[2])
+				}
+
 				elapsedTime := binary.LittleEndian.Uint32(record[2:])
```

Comment:
shouldn't this be shifted?

Reply:
🦅 nice catch. ^^thx. should be done here https://github.com/gnolang/gno/pull/4705/commits/a5aed74754fbef20f300250c87954b380ceb29e9